internal/auth/oidc/pam/provider.go

```diff
@@ -24,6 +25,192 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const loginFormTemplate = `<!DOCTYPE html>
```

Comment:
nit: can we  store the html as a file and use go:embed, I think it will save us from future formatting errors.

//go:embed templates/login_form.html
var loginFormTemplate string